dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/ContentletCacheImpl.java

```diff
@@ -58,6 +60,23 @@ public TranslatedQuery getTranslatedQuery(String key) {
 		}
 	}
 
+	public void addMetadataMap(final String key, final Map<String, Object> metadataMap) {
+
+		cache.put(META_DATA_MAP_KEY + key, UtilMethods.isSet(metadataMap)?
+				metadataMap:EMPTY_METADATA_MAP, metadataGroup);
+	}
+
+	public  Map<String, Object> getMetadataMap(final String key) {
+
+		Map<String, Object>  metadata = null;
+		try {
+			metadata=(Map<String, Object> )cache.get(META_DATA_MAP_KEY + key, metadataGroup);
+		} catch (DotCacheException e) {
+			Logger.debug(this, "Cache Entry not found", e);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "Cache Entry not found" appears 4 times in this file; the first occurrence is on line 105](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5547873)